pages/common/brew.md

```diff
@@ -0,0 +1,32 @@
+# brew
+
+> Package manager for macOS (or Linux).
+> More information: <https://brew.sh>.
+
+- Search for available formulae (i.e. packages) and casks (i.e. native packages):
+
+`brew search {{text}}`
+
+- Install the latest stable version of a formula or cask (use `--devel` for development versions):
+
+`brew install {{formula}}`
+
+- List all installed formulae and casks:
+
+`brew list`
+
+- Upgrade an installed formula or cask (if none is given, all installed formulae/casks are upgraded):
+
+`brew upgrade {{formula}}`
+
+- Fetch the newest version of Homebrew and of all formulae/casks from GitHub:
+
+`brew update`
+
+- Display information about a formula/cask (version, installation path, dependencies, etc.):
```

Comment:
Use the same expression everywhere: `formula/cask` or `formula or cask`?

Reply:
Not really sure.  I think both are fine, maybe @sbrl has a preference?

Reply:
I'd say if they are synonyms for each other, we should go with `formula/cask` and if they are two different things `formula or cask`.

Reply:
The latter is the case, so use `formula or cask`.

Reply:
Seems OK in ab99cf5b4f789d945082926dfc327e1f63bfab2c.